assets/scripts/streets/Geolocation.jsx

```diff
@@ -0,0 +1,156 @@
+/* global fetch */
+import React from 'react'
+import { connect } from 'react-redux'
+import { setSettings } from '../users/settings'
+import PropTypes from 'prop-types'
+import { Map, TileLayer, Marker, Popup } from 'react-leaflet'
+import SearchAddress from './SearchAddress'
+import {apiurlReverse, apikey} from './config'
+
+const OpenStreetMapTiles = 'http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png'
+const OpenStreetMapAttr = '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
```

Comment:
variable names should use `camelCase` or, in the case of constants like these, you should use `ALL_CAPS_WITH_UNDERSCORE`.